.travis.yml

```diff
@@ -11,4 +11,4 @@ script:
   - pub get
```

Comment:
While we're updating this repo, would you mind updating the list of dart versions at the top that we run this on?

```yaml
dart:
  - 2.4.1
  - stable
  - dev
```

That way we run on our lower bound, stable, and dev.